ajax/libs/react-transition-group/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-transition-group",
+  "filename": "react-transition-group.min.js",
+  "description": "A react component toolset for managing animations",
+  "keywords": [
+    "react",
+    "transition",
+    "addons",
+    "transition-group",
+    "animation",
+    "css",
+    "transitions"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https@github.com:reactjs/react-transition-group.git"
```

Comment:
Via GitHub website because had too much trouble trying to do normal git cloning :)

* Author updated
* URL updated.

Would you prefer I close and do a new PR with commit message since I did it via GitHub?